docker-compose.yml

```diff
@@ -102,6 +102,7 @@ services:
     build: dev-tools
     env_file: .env
     volumes:
+      - /app/dev-tools/
```

Comment:
This confused me for a bit, I think a comment here would be helpful for future me's :slightly_smiling_face: 

Maybe something like this, if I'm understanding correctly:
```
# Create an empty volume for `/app/dev-tools` so that node_modules is isolated to the container
```

Reply:
I thought this line was going to "mask" _dev-tools_ from the working directory, and show the _dev-tools_ from the image (i.e., containing the dependencies). But if /app/dev-tools/node_modules is going to be empty, it seems to me it defeats the purpose of creating the image in the first place.

(I read Docker docs for a bit: it would be great if "dev-tools" subdirectory could be skipped when mounting /app, but I didn't see a way

Reply:
> I thought this line was going to "mask" dev-tools from the working directory, and show the dev-tools from the image

It seems it does that!, and I mistyped something; sorry for the noise.